mods/d2k/rules/defaults.yaml

```diff
@@ -225,6 +227,11 @@
 			Rough: 80
 	Voiced:
 		VoiceSet: InfantryVoice
+	WithParachute:
+		UpgradeTypes: parachute
+		UpgradeMinEnabledLevel: 1
+		ShadowImage: parach-shadow
+		ShadowSequence: idle
```

Comment:
Godamn it. What do you guys use for SHP editing? I've downloaded this one program and I can't get anything done with it properly